packages/web-components/src/components/ic-switch/ic-switch.stories.mdx

```diff
@@ -159,3 +169,31 @@ import readme from "./readme.md";
     </form>`}
   </Story>
 </Canvas>
+
+### Playground
```

Comment:
Please could you add this playground in the React stories as well?

Reply:
Added!